string object_name
---
float64[] object_vertices
float64[] object_radii